Automattic/wp-calypso
# Jetpack Blocks: add index.js for each block

In this change we are refactoring editor.js files into tow files ediort.js and index.js. 
editor.js is now responsible for just registering the block and index.js is there to define all the blocks. 

This makes it easier to separate the code that defines the block from calling it. Which provides us with the flexibility in terms of when we call the register block code. 

#### Testing instructions

Make sure all the blocks still work as expetected in calypso
Create a jetpack bundle. Does the jetpack side still work as expected? 
Does the .com wp-admin side still work as expected.
